package.json

```diff
@@ -53,25 +53,23 @@
     "lint-report": "eslint --ext js,ts -f json -o eslint-report.json \"{common,extensions}/**/*.{js,ts}\"",
     "test-build": "./node_modules/.bin/gulp build:test",
     "test": "./node_modules/.bin/jest",
+    "test-ci": "./node_modules/.bin/jest --coverage",
     "ts-check": "./node_modules/.bin/tsc --noEmit",
     "validate": "npm run lint && npm run ts-check && npm run format-check && npm run test",
-    "validate-ci": "npm run lint-report && npm run ts-check && npm run format-check && npm run test --ci --coverage --runInBand"
+    "validate-ci": "npm run lint-report && npm run ts-check && npm run format-check && npm run test-ci"
   },
   "engines": {
     "node": ">=6"
   },
   "jest": {
     "clearMocks": true,
     "collectCoverageFrom": [
-      "**/*.ts"
+      "{common}**/*.ts"
```

Comment:
You might want to update your `sonar.coverage.exclusions` setting for the sonar-scanner in `config/utils.js` to ignore things that are in the folder `extensions`

Reply:
And done as well, thanks !